tests/vmi_networking_test.go

```diff
@@ -749,42 +749,47 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			Expect(err).ToNot(HaveOccurred())
 			Expect(len(serverVMI.Status.Interfaces)).To(Equal(1))
 
-			serverIP := serverVMI.Status.Interfaces[0].IP
-			if netutils.IsIPv6String(serverIP) {
-				By("Checking traceroute from vmi to cluster nodes gateway")
-				// Cluster nodes subnet (docker network gateway)
-				// Docker network subnet cidr definition https://bit.ly/2wZTgMK
-				err := tests.CheckForTextExpecter(serverVMI, createExpectTraceroute6("2001:db8:1::1"), 30)
-				Expect(err).ToNot(HaveOccurred(), "Failed to traceroute to VMI %s within the given timeout", serverVMI.Name)
-			} else {
-				if ipv4NetworkCIDR == "" {
-					ipv4NetworkCIDR = api.DefaultVMCIDR
+			for _, serverIP := range serverVMI.Status.Interfaces[0].IPs {
+				if serverIP == "" {
```

Comment:
after refactor of the vm PR we wont have this if,
but the skip will be under if ipv6 condition, since this way it it skip exactly what needed and be explicit that it doesnt skip IPv4.
the skip it just until the core changes of the dual stack are fixed anyway.